index.html

```diff
@@ -36,6 +36,186 @@ <h1>GLAD TO SEE <br>YOU!</h1>
             <li><a href="#"><i class="fab fa-github"></i></a></li> 
         </ul>
       </div>
```

Comment:
OPTIONAL

- [ ] When you implemented the projects and about sections, bug were introduced that affected the design of the section above. You may need to give it a fix.
![Screenshot from 2021-09-20 14-41-53](https://user-images.githubusercontent.com/65501535/133996656-b01d6dbc-9ea8-4a93-898a-c46f3b042868.png)
.